accounts/accounts.go

```diff
@@ -0,0 +1,54 @@
+package accounts
+
+import "errors"
+
+// Account : State of a particular account
+type Account struct {
+	Balance float64
```

Comment:
I don't think golang has operator overload. You'll have to make typeactions like 

```
func AddDollars(a account, value Currency) {
    a.dollars += value.dollars
    a.cents += value.cents
}
```